src/models/github-server-app.ts

```diff
@@ -175,7 +175,9 @@ export class GitHubServerApp extends EncryptedModel {
 
 	}
 
-	static async findForUuid(uuid: string): Promise<GitHubServerApp | null> {
+	static async findForUuid(uuid: string | undefined): Promise<GitHubServerApp | null> {
+		if (!uuid) return null;
```

Comment:
Added this because the where clause fails if you pass an undefined value (as may happen for cloud path)